src/app/component/general/eco-news/create-news/create-news.component.html

```diff
@@ -1,68 +1,5 @@
-<div class="container">
```

Comment:
revert this file, it's out of the scope of your ticket and this functionality(news form) isn't done yet for implementation buttons styles

Reply:
fixed, this task was 22 hours ago